qiskit/visualization/timeline/interface.py

```diff
@@ -361,6 +365,14 @@ def draw(
     temp_style = stylesheet.QiskitTimelineStyle()
     temp_style.update(style or stylesheet.IQXStandard())
 
+    if target is None:
+        warnings.warn(
+            "Target is not specified in Qiskit 2.0.0 this will be required to get the duration of "
+            "instructions.",
```

Comment:
```suggestion
            "Target is not specified. In Qiskit 2.0.0 this will be required to get the duration of "
            "instructions.",
```